prime-router/docs/getting-started/standard-operating-procedures/changing-transforms.md

```diff
@@ -93,6 +93,7 @@ Each element contains the following properties, listed in order of execution:
   constants defined at their schema level and will overwrite any that have the same name.
   - A reserved constant, `%resource`, is automatically provided, which maps to the focus resource
     (generally, the `resource` property).
+  - A reserved constant `%context` is provided and maps to the previous `%resource` value
```

Comment:
I could see this being a little unclear, maybe something like:
```suggestion
  - A reserved constant `%context` is provided and maps to the previous `%resource` value when a new `resource` value is also being set in the same element.
```
Depending on whether `context` can be used anytime, or only when `resource` is set, that suggestion may still be misleading, haven't gotten far enough into the PR to know one way or the other.

Reply:
Context is actually always set when evaluating a FHIR path.  When starting at the root, focusResource, rootResource and context will all be the same value

Reply:
Ah gotcha, "previous `%resource`" is a little confusing especially for cases when a new `resource` isn't being set, but I'm not sure how else to put it, so I think we're okay here.